cli/tests/test_repo_change_report.rs

```diff
@@ -224,3 +224,32 @@ fn test_report_conflicts_with_divergent_commits() {
     [EOF]
     ");
 }
+
+#[test]
+fn test_report_conflicts_with_resolving_conflicts_hint_disabled() {
+    let test_env = TestEnvironment::default();
+    test_env.run_jj_in(".", ["git", "init", "repo"]).success();
+    let repo_path = test_env.env_root().join("repo");
+
+    std::fs::write(repo_path.join("file"), "A\n").unwrap();
+    test_env.run_jj_in(&repo_path, ["commit", "-m=A"]).success();
+    std::fs::write(repo_path.join("file"), "B\n").unwrap();
+    test_env.run_jj_in(&repo_path, ["commit", "-m=B"]).success();
+    std::fs::write(repo_path.join("file"), "C\n").unwrap();
+    test_env.run_jj_in(&repo_path, ["commit", "-m=C"]).success();
+
+    let output = test_env.run_jj_in(&repo_path, ["rebase", "-s=description(B)", "-d=root()", "--config", "hints.resolving-conflicts=false"]);
```

Comment:
nit: you'll need to run `cargo fmt`, and `"--config=hints..."` will save vertical space.

Reply:
Strange, I did try putting it in a single argument and got an error. I just tried again and it's working fine. I think perhaps I missed the `=` the first time around. I've changed that and run `cargo fmt`